docs/CHANGELOG.rst

```diff
@@ -5,10 +5,18 @@ Change Log
 0.4.0
 ====================
 
-- Decoupled SOMD and removing run_somd.sh and template_config.cfg. Replaced with a3fe.configuration.slurm_config.SlurmConfig and a3fe.configuration.engine_config.SomdConfig. 
-- And system preparation saved as a readable yaml file. 
+**Note**: 
+This is a breaking change! Old pickle files are not compatible with this version. 
+If you need to load old pickle files, please revert to version 0.3.x.
+
+- Decoupled SOMD engine from a3fe calculation and removed run_somd.sh and template_config.cfg, replacing them with a3fe.configuration.slurm_config.SlurmConfig and a3fe.configuration.engine_config.SomdConfig. 
+- SystemPreparationConfigs saved as readable yaml file, rather than as a pickle. 
 - Separated system preparation configuration from system preparation and configured SOMD with a3fe.configuration.system_prep_config.SomdSystemPreparationConfig.
 
+**Migration Guide**:
+In previous versions, users needed to configure run_somd.sh and template_config.cfg files to run calculations. 
+However, with version 0.4.0, these files are no longer needed. Instead, configuration objects (SlurmConfig, SomdConfig, and SomdSystemPreparationConfig) are passed directly to the Calculation object.
```

Comment:
Brilliant, thanks! Maybe add a link to where the docs show an example of doing this?